pkg/cmd/openshift-apiserver/openshiftapiserver/config.go

```diff
@@ -215,22 +219,20 @@ func NewOpenshiftAPIConfig(openshiftAPIServerConfig *configapi.MasterConfig) (*O
 			SubjectLocator:                     subjectLocator,
 			LimitVerifier:                      imageLimitVerifier,
 			RegistryHostnameRetriever:          registryHostnameRetriever,
-			AllowedRegistriesForImport:         openshiftAPIServerConfig.ImagePolicyConfig.AllowedRegistriesForImport,
-			MaxImagesBulkImportedPerRepository: openshiftAPIServerConfig.ImagePolicyConfig.MaxImagesBulkImportedPerRepository,
+			AllowedRegistriesForImport:         newConfig.ImagePolicyConfig.AllowedRegistriesForImport,
+			MaxImagesBulkImportedPerRepository: newConfig.ImagePolicyConfig.MaxImagesBulkImportedPerRepository,
 			AdditionalTrustedCA:                caData,
 			RouteAllocator:                     routeAllocator,
 			ProjectAuthorizationCache:          projectAuthorizationCache,
 			ProjectCache:                       projectCache,
-			ProjectRequestTemplate:             openshiftAPIServerConfig.ProjectConfig.ProjectRequestTemplate,
-			ProjectRequestMessage:              openshiftAPIServerConfig.ProjectConfig.ProjectRequestMessage,
+			ProjectRequestTemplate:             newConfig.ProjectConfig.ProjectRequestTemplate,
+			ProjectRequestMessage:              newConfig.ProjectConfig.ProjectRequestMessage,
 			ClusterQuotaMappingController:      clusterQuotaMappingController,
 			RESTMapper:                         restMapper,
 			SCCStorage:                         sccStorage,
+			ServiceAccountMethod:               newConfig.ServiceAccountOAuthGrantMethod,
 		},
 	}
-	if openshiftAPIServerConfig.OAuthConfig != nil {
-		ret.ExtraConfig.ServiceAccountMethod = openshiftAPIServerConfig.OAuthConfig.GrantConfig.ServiceAccountMethod
-	}
```

Comment:
why is this gone?

Reply:
ic, moved up

Reply:
> why is this gone?

two lines up

`ServiceAccountMethod:               newConfig.ServiceAccountOAuthGrantMethod,`

https://github.com/openshift/origin/pull/20787/files/355c3e75e7c129135f9e79fa99132f0d512ee3d0#diff-6631c206d2fb778961dcfa92e2f0f7caR233